### PR TITLE
add matrix_nms in python/paddle/vision/ops.py

### PR DESCRIPTION
**Pull request by ghostxsl**
### PR types

New features
### PR changes

APIs
### Describe

add matrix_nms in python/paddle/vision/ops.py
预览链接
http://10.136.157.23:8090/documentation/docs/zh/faq/index_cn.html?reviewVersion=jenkins-doc-review-2388

### CODE REVIEW

File: python/paddle/vision/ops.py [hunk 1]
```diff
@@ -1540,3 +1530,140 @@ def _nms(boxes, iou_threshold):
...
+        return_index(bool): Whether return selected index. Default: False
+        return_rois_num(bool): whether return rois_num. Default: True
+        name(str): Name of the matrix nms op. Default: None.
+    Returns:
+        A tuple with three Tensor: (Out, Index, RoisNum) if return_index is True,
+        otherwise, a tuple with two Tensor (Out, RoisNum) is returned.
+        Out (Tensor): A 2-D Tensor with shape [No, 6] containing the
+             detection results.
+             Each row has 6 values: [label, confidence, xmin, ymin, xmax, ymax]
+             (After version 1.3, when no boxes detected, the lod is changed
```
Comment (jerrywgz) [inline python/paddle/vision/ops.py]: 这句说明删掉吧

Comment (ghostxsl) [reply python/paddle/vision/ops.py]: done, thx~

**Review by jerrywgz — DISMISSED** [review]

**Review by XiaoguangHu01 — DISMISSED** [review]
lgtm

**Review by XiaoguangHu01 — DISMISSED** [review]
LGTM

**Review by heavengate — DISMISSED** [review]

File: python/paddle/vision/ops.py [hunk 2]
```diff
@@ -1802,3 +1791,138 @@ def generate_proposals(scores,
...
+               keep_top_k,
+               use_gaussian=False,
+               gaussian_sigma=2.,
+               background_label=0,
+               normalized=True,
+               return_index=False,
+               return_rois_num=True,
+               name=None):
+    """
+    **Matrix NMS**
```
Comment (dingjiaweiww) [inline python/paddle/vision/ops.py]: 这个可以去掉

Comment (ghostxsl) [reply python/paddle/vision/ops.py]: done, thx~

**Review by dingjiaweiww — APPROVED** [review]

**Review by XiaoguangHu01 — APPROVED** [review]
LGTM

**Review by heavengate — APPROVED** [review]

**Pull request merged**